### ISSUE TITLE
SHA1/MD5 Digest for v7.6.3 actually SHA1/SHA256

### ISSUE THREAD

**Issue by SilmorSenedlen** [notepad-plus-plus/notepad-plus-plus#5274]
Goog day.
I noticed that SHA1/MD5 digest link([https://notepad-plus-plus.org/repository/7.x/7.6.3/npp.7.6.3.sha1.md5.digest.txt](url)) actually contains SHA1/SHA256 hash.
Need to correct hash names in digest and download page.

Will you be using a new pair of hash algo from this release?
That would be great.

P.S. Will you add SHA256 digital signature for Windows app/installer binaries?

**Comment by donho** [comment]
> Will you add SHA256 digital signature for Windows app/installer binaries?

Do you think it's helpful?

**Comment by Xant1k** [comment]
> Will you add SHA256 digital signature for Windows app/installer binaries?

SHA-512, BLAKE2bp/sp best choice.

**Comment by SilmorSenedlen** [comment]
> 
> 
> > Will you add SHA256 digital signature for Windows app/installer binaries?
> 
> Do you think it's helpful?

At the moment, apparently, this is not necessary(except for drivers), but SHA1 signatures will definitely be deprecated in the future.
I did not find specific deadlines for that case on fast search, but if the addition of the second signature does not require material costs and is not too time-consuming, it will not be excessive.
[https://social.technet.microsoft.com/wiki/contents/articles/32288.windows-enforcement-of-sha1-certificates.aspx](url)

**Comment by SilmorSenedlen** [comment]
> 
> 
> > Will you add SHA256 digital signature for Windows app/installer binaries?
> 
> SHA-512, BLAKE2bp/sp best choice.

Yes, but I think that they are not standardized as Digital Signatures for binary signing in Windows.

The best choice for tech-savy users will be GPG-signed distributed files(not only binaries, but zip as well) by creators GPG private key(aka RSA 4096).
